069-shared-volume-mount.md

```diff
@@ -0,0 +1,108 @@
+# Shared Volume Mount Changes
+
+The proposed update to the Strimzi Kafka Operator introduces management of shared volume mounts for kafka connect cluster, 
```

Comment:
We need to have a reusable solution that covers all the different use cases for mounting some volumes. So you should not focus just on your immediate need. We have to make sure this change can survive the _next proposal_ from someone else with different requirement.

Reply:
I am with Jakub on this.